server/api/v1.py

```diff
@@ -564,6 +564,29 @@ def rate_review_for_user():
 
     return vote_added_response
 
+@api.route('/user/remove_scholarship_from_profile', methods=['PUT'])
```

Comment:
Suggestion for a more RESTful API endpoint:

```
@api.route('/user/scholarships/<scholarship_id>', methods=['DELETE'])
```

These resources have influenced how I've thought about RESTful API endpoints:

- http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api?hn
- https://developer.github.com/v3/